Changelogs for Digital Twin Manager UI 1022.0.9 (#3797)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1022-9-2-auto-generated-property-keys-now-retain-the-original-casing-in-the-asset-properties-screen.md

---
date: ""

> title: "Auto-generated property keys now retain the original casing in the Asset Properties screen"

Reviewer: ```suggestion
title: "Auto-generated property keys now retain the original casing"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: ""
version: "1022.9.2"
---

> Previously, when creating a property, the auto-generated key converted

Reviewer: ```suggestion
Previously, when creating an asset property, the auto-generated key converted
```

> all letters in the label to lowercase and replaced special characters

Reviewer: ```suggestion
all letters in the label in the **Asset properties** page to lowercase and replaced special characters
```
